src/main/webapp/app/entities/promotion/promotion-dialog.html

```diff
@@ -16,8 +16,7 @@ <h4 class="modal-title" id="myPromotionLabel" data-translate="crmisticApp.promot
         <div class="form-group">
             <label class="control-label" data-translate="crmisticApp.promotion.annee" for="field_annee">Annee</label>
                 <div class="input-group">
-                    <input id="field_annee" type="text" class="form-control" name="annee" datetime-picker="{{dateformat}}" ng-model="vm.promotion.annee" is-open="vm.datePickerOpenStatus.annee"
-                    required  />
+                    <input id="field_annee" type="text" class="form-control" name="annee" datepicker-mode="'year'" datetime-picker="yyyy" ng-model="vm.promotion.annee" is-open="vm.datePickerOpenStatus.annee" enable-time="false" datepicker-options="vm.datepickerOptions" />
```

Comment:
You can delete datepicker-mode="'year'". It is redundant with the option givent in the controller